src/components/services-view/services-view.html

```diff
@@ -44,5 +45,16 @@
         </div>
       </div>
     </div>
+
+    <div ng-if="$ctrl.currentFilter === 'other'"
+         class="sub-category-row other"
+         pf-card-view config="$ctrl.cardViewConfig" items="$ctrl.filteredItems">
+      <div class="card-container">
+        <div ng-if="!item.imageUrl" class="card-icon {{item.iconClass}}"></div>
+        <div ng-if="item.imageUrl" class="card-img"><img src="{{item.imageUrl}}"></div>
+        <div class="card-name">{{item.name}}</div>
+      </div>
+    </div>
+
   </div>
```

Comment:
We want to hide/skip the sub-categorie rows for the 'Other' main category and apply additional margin at top.  I did have it working without this extra block but it seemed overly complicated.  I'll look into it some more.